### PR TITLE
Simplify decrypt intent creation

### PR DESCRIPTION
**Pull request by GeorgCantor**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## :scroll: Description

Simplified code using operator `apply`

## :bulb: Motivation and Context

Following the trends of the Kotlin language

## :pencil: Checklist

- [ ] I formatted the code with the IDE's reformat action (Ctrl + Shift + L/Cmd + Shift + L)
- [ ] I reviewed submitted code
- [ ] I added a [CHANGELOG](CHANGELOG.md) entry if applicable

### CODE REVIEW

**Review by msfjarvis — APPROVED** [review]

**Comment by msfjarvis** [comment]
Thanks!

File: app/src/main/java/com/zeapo/pwdstore/LaunchActivity.kt [hunk 1]
```diff
@@ -41,12 +41,13 @@ class LaunchActivity : AppCompatActivity() {
...
-            decryptIntent.putExtra("FILE_PATH", intent.getStringExtra("FILE_PATH"))
-            decryptIntent.putExtra("REPO_PATH", intent.getStringExtra("REPO_PATH"))
-            decryptIntent.putExtra("LAST_CHANGED_TIMESTAMP", intent.getLongExtra("LAST_CHANGED_TIMESTAMP", 0L))
-            startActivity(decryptIntent)
+            Intent(this, DecryptActivity::class.java).apply {
+                putExtra("NAME", intent.getStringExtra("NAME"))
+                putExtra("FILE_PATH", intent.getStringExtra("FILE_PATH"))
+                putExtra("REPO_PATH", intent.getStringExtra("REPO_PATH"))
+                putExtra("LAST_CHANGED_TIMESTAMP", intent.getLongExtra("LAST_CHANGED_TIMESTAMP", 0L))
+                startActivity(this)
```
Comment (fmeum) [inline app/src/main/java/com/zeapo/pwdstore/LaunchActivity.kt:49]: Let's move this out of the apply to make it easier to parse what the intent ends up being used for.

**Review by fmeum — CHANGES_REQUESTED** [review]

**Comment by fmeum** [comment]
Well, nvm, I'm just not a fast enough reviewer :-D

File: app/src/main/java/com/zeapo/pwdstore/LaunchActivity.kt (hunk 1 above)
Comment (msfjarvis) [reply app/src/main/java/com/zeapo/pwdstore/LaunchActivity.kt:49]: It looked neat for a second so I let it pass but yeah its better to keep it outside. I've added that to #1068.

**Pull request merged**